jetbrains: 2025.1 -> 2025.1.3

jamesward opened the pull request:
```
jetbrains: 2025.1 -> 2025.1.3
    
jetbrains.clion: 2025.1.1 -> 2025.1.2
jetbrains.datagrip: 2025.1.2 -> 2025.1.3
jetbrains.dataspell: 2025.1 -> 2025.1.1
jetbrains.gateway: 2025.1.1 -> 2025.1.2
jetbrains.goland: 2025.1.1 -> 2025.1.2
jetbrains.idea-community: 2025.1.1.1 -> 2025.1.2
jetbrains.idea-ultimate: 2025.1.1.1 -> 2025.1.2
jetbrains.phpstorm: 2025.1.1 -> 2025.1.2
jetbrains.pycharm-community: 2025.1.1 -> 2025.1.2
jetbrains.pycharm-professional: 2025.1.1 -> 2025.1.2
jetbrains.ruby-mine: 2025.1.1 -> 2025.1.2
jetbrains.rust-rover: 2025.1.2 -> 2025.1.3
jetbrains.webstorm: 2025.1.1 -> 2025.1.2
```

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

JBassett commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 413926`
Commit: `42b8efbf490d9887c7858625b4bbe6ac1e7fb110`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.pycharm-community-bin</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rust-rover</li>
  </ul>
</details>

jamesward commented:
Just waiting on a merge. But since this sat a bit, I'm re-running the update script to catch some new releases. Will push that shortly.

jamesward committed:
jetbrains: 2025.1 -> 2025.1.3

jetbrains.clion: 2025.1.1 -> 2025.1.2
jetbrains.datagrip: 2025.1.2 -> 2025.1.3
jetbrains.dataspell: 2025.1 -> 2025.1.1
jetbrains.gateway: 2025.1.1 -> 2025.1.2
jetbrains.goland: 2025.1.1 -> 2025.1.2
jetbrains.idea-community: 2025.1.1.1 -> 2025.1.2
jetbrains.idea-ultimate: 2025.1.1.1 -> 2025.1.2
jetbrains.phpstorm: 2025.1.1 -> 2025.1.2
jetbrains.pycharm-community: 2025.1.1 -> 2025.1.2
jetbrains.pycharm-professional: 2025.1.1 -> 2025.1.2
jetbrains.ruby-mine: 2025.1.1 -> 2025.1.2
jetbrains.rust-rover: 2025.1.2 -> 2025.1.3
jetbrains.webstorm: 2025.1.1 -> 2025.1.2

jamesward committed:
jetbrains.plugins: update

winterqt reviewed (approved):
Thanks! Tested and working on x86_64-linux.
